### DISCUSSION TITLE
Recommended AWS EKS cluster size for Milvus with DISKANN Index Type

### DISCUSSION THREAD

**Discussion by akmalmasud96** [milvus-io/milvus#29054] [Q&A and General discussion] [answered]
I want to create a cluster with DiskANN index-type, I am inquiring about the size of query-nodes. However, there is no estimation in the tool of https://milvus.io/tools/sizing/ for the index-type of DiskANN. 
I have 200M records with 512 embedding_size

What should be the approximate query-node size be and other nodes configurations be, and if there is any tool, please point towards it.

**Comment by yhmo** [comment]
The size of DiskANN is a bit smaller than IVF_SQ8. You can use the IVF_SQ8 to estimate.

**Reply by akmalmasud96** [reply]
Thanks, 
Can you also please illustrate how should the disk-space be selected?

**Comment by yhmo** [comment]
![Screenshot from 2023-12-08 10-32-11](https://github.com/milvus-io/milvus/assets/2282099/4aac4639-f6c7-441f-9b77-d4254aa3603b)

**Reply by akmalmasud96** [reply]
Thanks [yhmo](https://github.com/yhmo) , for sharing the detail. 
These calculations are for the raw data storage. However, for the index data, what and how should be the calculations for the query-nodes and index-nodes?
In the documentation, following instructions are described for index and query node:

> For a Milvus Cluster instance, the data path should be /var/lib/milvus/data in the containers where the QueryNodes and IndexNodes run.

**Reply by yhmo** [reply, marked as answer]
Index node: 10GB disk space for each node
Query node: 150% of "Raw file size" totally, average allocated to each node. For example, "Raw file size" = 300GB, there are 5 query nodes in the cluster, each query node assigned (300*150%)/5=18Gb disk space
